lib/database.js

```diff
@@ -29,8 +42,14 @@ const buildWhere = (conditions, firstArgIndex = 1) => {
     const keys = Object.keys(where);
     for (const key of keys) {
       const [operator, value] = whereValue(where[key]);
-      conjunction.push(`"${key}" ${operator} $${i++}`);
-      args.push(value);
+      if (value instanceof Array) {
+        const list = value.map(() => `$${i++}`).join(',');
```

Comment:
test fails at this line because you can't define function inside loop, so please rewrite using external call or nested loop